skrifa/src/glyph_name.rs

```diff
@@ -93,6 +93,10 @@ impl<'a> GlyphNames<'a> {
                 .and_then(|sid| GlyphName::from_cff_sid(cff, sid)),
             _ => None,
         };
+        // If name is empty string, synthesize it
+        if name.as_ref().map_or(true, |s| s.is_empty()) {
```

Comment:
```suggestion
        if name.as_ref().is_none_or(|s| s.is_empty()) {
```
Apparently not quite idiomatic enough for clippy :)